docs/security/non-events.md

```diff
@@ -0,0 +1,84 @@
+<!--[metadata]>
++++
+aliases = ["/engine/articles/security/"]
```

Comment:
nice catch... some copy pasta


On Sun, May 8, 2016 at 3:13 PM, Sebastiaan van Stijn <
notifications@github.com> wrote:

> In docs/security/non-events.md
> <https://github.com/docker/docker/pull/22579#discussion_r62439448>:
>
> > @@ -0,0 +1,84 @@
> > +<!--[metadata]>
> > ++++
> > +aliases = ["/engine/articles/security/"]
>
> This should probably be removed for this page [image: :smile:]
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/22579/files/c63f734f5b974c3214225fb9faf0a1cef1c91bbe#r62439448>
>



-- 


Jessie Frazelle
4096R / D4C4 DD60 0D66 F65A 8EFC  511E 18F3 685C 0022 BFF3
pgp.mit.edu <http://pgp.mit.edu/pks/lookup?op=get&search=0x18F3685C0022BFF3>